### DISCUSSION TITLE
plotting slower than expected

### DISCUSSION THREAD

**Discussion by MNevittAgile** [swar/Swar-Chia-Plot-Manager#767] [Q&A]
System:

SSD:  2TB Sabrent Rocket 4 Plus NVMe (up to 7000 MB/s (read) and 5300 MB/s (write))
RAM:  64 Gig of DDR4 4000
Processor:  Ryzen 3900X (12 core / 24 Thread)
Motherboard: Asus TUF B550-plus

Okay, let me first say that I'm also running an RTX 3070 mining ETH on the same system.  I didn't think this would affect things, but maybe it is more than I think.  That said, here are my pertinent settings:

threads: 3
memory buffer: 5000
max concurrent: 10
max with early start: 11
stagger minutes: 45
max for phase 1: 5
process priority 32

plots are taking  12-13 hours each.   I expected it to be half that with such a fast drive.  Am I just trying to run too many through that one drive?  

I also have a SATA drive that I tried to break up the work being on all one drive, but that didn't help.  I assume that having the two drives running was just competing for bus speed or something.  I'm waiting on another NVMe drive to see if running that at the same time will be better than splitting with the SATA, as I have two M.2 slots.  Currently the 2nd one is for the OS and is only a 256 gig drive.

Any idea what my bottleneck is?

**Comment by bebanowski** [comment]
I think you are running too many for one drive, as you rightly stated. The read/write speeds on your SSD will [drop rapidly](https://www.howtogeek.com/165542/why-solid-state-drives-slow-down-as-you-fill-them-up/#:~:text=Fill%20your%20solid%2Dstate%20drive,and%20NAND%20Flash%20storage%20work.&text=A%20nearly%20full%20solid%2Dstate,operations%2C%20slowing%20down%20your%20computer.) once space is used up, so cramming it with too many plots at once will greatly reduce plotting speeds. Phase 1 of plotting is also the most resource intensive, so it is a bad idea to have 5 in phase 1 at any one time with your set up. I would suggest purchasing an additional 2TB SSD for your build as this is clearly your major bottleneck.

**Reply by MNevittAgile** [reply]
Okay, thank you very much for the reply. :)
